### PR TITLE
Added support for the 1.12 series.

### PR DESCRIPTION
**Pull request by carlonluca**

### CODE REVIEW

File: v1.12/Dockerfile.amd64 [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  apk add nodejs npm && \
+  git clone -b ${GITEA_BRANCH} ${GITEA_REPO} /srv/app/src/${GITEA_PATH} && \
+  cd /srv/app/src/${GITEA_PATH} && \
+  TAGS="bindata sqlite" make build install
+
+FROM webhippie/alpine:latest-amd64
+
+LABEL maintainer="Thomas Boerger <thomas@webhippie.de>" \
+  org.label-schema.name="Gitea" \
+  org.label-schema.version="1.11" \
```
Comment (tboerger) [inline v1.12/Dockerfile.amd64:18]: The label version has not been increased

File: v1.12/Dockerfile.arm32v6 [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  apk add nodejs npm && \
+  git clone -b ${GITEA_BRANCH} ${GITEA_REPO} /srv/app/src/${GITEA_PATH} && \
+  cd /srv/app/src/${GITEA_PATH} && \
+  TAGS="bindata sqlite" make build install
+
+FROM webhippie/alpine:latest-arm32v6
+
+LABEL maintainer="Thomas Boerger <thomas@webhippie.de>" \
+  org.label-schema.name="Gitea" \
+  org.label-schema.version="1.11" \
```
Comment (tboerger) [inline v1.12/Dockerfile.arm32v6:18]: Same here

File: v1.12/Dockerfile.arm64v8 [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  apk add nodejs npm && \
+  git clone -b ${GITEA_BRANCH} ${GITEA_REPO} /srv/app/src/${GITEA_PATH} && \
+  cd /srv/app/src/${GITEA_PATH} && \
+  TAGS="bindata sqlite" make build install
+
+FROM webhippie/alpine:latest-arm64v8
+
+LABEL maintainer="Thomas Boerger <thomas@webhippie.de>" \
+  org.label-schema.name="Gitea" \
+  org.label-schema.version="1.11" \
```
Comment (tboerger) [inline v1.12/Dockerfile.arm64v8:18]: Same here

File: v1.12/manifest.tmpl [hunk 4]
```diff
@@ -0,0 +1,19 @@
+image: webhippie/gitea:1.11
```
Comment (tboerger) [inline v1.12/manifest.tmpl:1]: Within this file all 1.11 references got to be replaced by 1.12

**Comment by carlonluca** [comment]
References should be updated now.

**Comment by dschmidt** [comment]
You forgot to add 1.12 to the versions matrix in .drone.star

**Comment by tboerger** [comment]
Damn, totally missed that 😂

**Comment by tboerger** [comment]
I have created https://github.com/dockhippie/gitea/pull/25to get it finally built.

**Pull request merged**